projects/driver_controls/test/test_ads1015.c

```diff
@@ -0,0 +1,60 @@
+#include <stdint.h>
+#include <stdio.h>
+#include "ads1015.h"
+#include "unity.h"
+
+#include "gpio.h"
+#include "gpio_it.h"
+#include "i2c.h"
+#include "i2c_mcu.h"
+#include "interrupt.h"
+#include "status.h"
+
+static ADS1015Data data;
+GPIOAddress ready_pin = { GPIO_PORT_B, 2 };
+
+static I2CSettings i2c_settings = {
+  .speed = I2C_SPEED_FAST, .scl = { GPIO_PORT_B, 10 }, .sda = { GPIO_PORT_B, 11 }
```

Comment:
Please add `//` to fix your formatting. Also, we generally prefer to use designated initializers.